saleor/graphql/product/mutations/products.py

```diff
@@ -334,6 +372,29 @@ class Meta:
         description = 'Updates an existing product.'
         model = models.Product
 
+    @classmethod
+    def clean_sku(cls, product_type, cleaned_input, errors):
+        pass
```

Comment:
Let's add a comment here as well to explain why we explicitly skip this logic. E.g: "SKU is an optional field in the product update mutation, so we explicitly skip the validation."